### PR TITLE
add asking gpt endpoint to send a request to openAi api

### PR DESCRIPTION
**Pull request by romaissaMe**

### CODE REVIEW

**Comment by romaissaMe** [comment]
@achab

File: app/api/routers/ask_gpt.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import openai
+from fastapi import APIRouter
+from dotenv import load_dotenv
+
+load_dotenv()
+ask_router=APIRouter()
+openai.api_key=os.getenv("OPENAI_API_KEY")
+
+@ask_router.get('/ask_gpt')
+def ask():
```
Comment (achab) [inline app/api/routers/ask_gpt.py:11]: please add a parameter `question` and ask this question to ChatGPT

File: app/api/routers/ask_gpt.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import os
+import openai
+from fastapi import APIRouter
+from dotenv import load_dotenv
+
+load_dotenv()
+ask_router=APIRouter()
+openai.api_key=os.getenv("OPENAI_API_KEY")
+
+@ask_router.get('/ask_gpt')
```
Comment (achab) [inline app/api/routers/ask_gpt.py:10]: please name the endpoint `/ask`

**Comment by romaissaMe** [comment]
@achab

**Comment by achab** [comment]
Great addition! Could you add a section to the README to explain how to do API calls. Please use Markdown formatting option. Like:

Commands to use the API:
- `/ask` endpoint
  ```bash
  something # command on windows
  something # command on mac/linux
  ```
- `/anonymize` endpoint
  ```bash
  something # command on windows
  something # command on mac/linux
  ```

File: README.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ Running unit tests on anonymize.py using pytest
...
 ## To run the API
 uvicorn app.api.main:app --reload  
-### To anonymize a file run the following command 
-In windows: **curl.exe -X POST -F "file=@tests\example.pdf" http://localhost:8000/anonymize**  
-The equivalent command for linux/mac: **curl http://localhost:8000/anonymize -F "file=@tests/example.pdf"**
+## Commands To Use The API  
+- /anomymize endpoint  
+  **curl http://localhost:8000/anonymize -F "file=@tests/example.pdf"**         
+- /ask endpoint  
+  **curl -X GET "http://127.0.0.1:8000/ask?question=your_question" -H "accept: application/json"**
```
[the remark this answers is not in the document]
Comment (achab) [reply README.md]: and then headers option is not mandatory, so you can remove it

File: README.md [hunk 4]
```diff
@@ -9,6 +9,8 @@ Running unit tests on anonymize.py using pytest
 pip install -r requirements.txt
 ## To run the API
 uvicorn app.api.main:app --reload  
-### To anonymize a file run the following command 
-In windows: **curl.exe -X POST -F "file=@tests\example.pdf" http://localhost:8000/anonymize**  
-The equivalent command for linux/mac: **curl http://localhost:8000/anonymize -F "file=@tests/example.pdf"**
+## Commands To Use The API  
+- ### /anomymize endpoint  
+  curl http://localhost:8000/anonymize -F "file=@tests/example.pdf"
```
Comment (achab) [inline README.md:14]: can you add write it like that? same for the command below

```bash
curl http://localhost:8000/anonymize -F "file=@tests/example.pdf"
```

File: README.md [hunk 5]
```diff
@@ -9,6 +9,8 @@ Running unit tests on anonymize.py using pytest
...
 ## To run the API
 uvicorn app.api.main:app --reload  
-### To anonymize a file run the following command 
-In windows: **curl.exe -X POST -F "file=@tests\example.pdf" http://localhost:8000/anonymize**  
-The equivalent command for linux/mac: **curl http://localhost:8000/anonymize -F "file=@tests/example.pdf"**
+## Commands To Use The API  
+- ### /anomymize endpoint  
+  curl http://localhost:8000/anonymize -F "file=@tests/example.pdf"        
+- ### /ask endpoint  
+  curl "http://127.0.0.1:8000/ask?question=your_question"
```
Comment (achab) [inline README.md:16]: `your-question` instead of `your question`. there's no whitespace in a url

**Review by achab — APPROVED** [review]
Great work

File: README.md (hunk 5 above)
Comment (romaissaMe) [reply README.md:16]: it's written your_question

Comment (achab) [reply README.md:16]: ok my bad, the whole url is underlined

**Pull request merged**